stingray/modeling/posterior.py

```diff
@@ -333,6 +333,33 @@ def evaluate(self, pars, neg=False):
         else:
             return loglike
 
+class PSDLogLikelihoodBound(PSDLogLikelihood):
+    def __init__(self, *args, **kwargs):
```

Comment:
The call to `_fitter_to_model_params`. Here I needed the fitter to see all parameters